File: unsorted/is_palindrome.py
def is_palindrome(x):
    """
    copyright: https://leetcode.com/problems/palindrome-number/submissions/
    Runtime: 56 ms, faster than 79.65% of Python3 online submissions for Palindrome Number.
    Memory Usage: 14.1 MB, less than 91.87% of Python3 online submissions for Palindrome Number.
    """
    num = str(x)
    return num == num[::-1]

File: unsorted/is_palindrome.rs
///https://leetcode.com/problems/palindrome-number/submissions/
/// Runtime: 8 ms, faster than 58.61% of Rust online submissions for Palindrome Number.
/// Memory Usage: 2.1 MB, less than 25.83% of Rust online submissions for Palindrome Number.
pub fn is_palindrome(x: i64) -> bool {
    let value: String = x.to_string();
    let reversed: String = value.chars().rev().collect();
    value == reversed
}
#[cfg(test)]
mod tests {
    use super::is_palindrome;
    #[test]
    fn test_is_palindrome() {
        assert_eq!(is_palindrome(-121), false);
        assert_eq!(is_palindrome(121), true);
        assert_eq!(is_palindrome(10), false);
    }
}
